src/modules/neo4j/neo4j-container.ts

```diff
@@ -0,0 +1,68 @@
+import { GenericContainer, Wait } from "../..";
+import { Image, Tag } from "../../repo-tag";
+import { Host } from "../../docker-client-factory";
+import { DockerClient } from "../../docker-client";
+import { BoundPorts } from "../../bound-ports";
+import { StartedTestContainer } from "../../test-container";
+
+export interface StartedNeo4jTestContainer extends StartedTestContainer {
+  getPassword: () => string;
+  getUsername: () => string;
+  getBoltUri: () => string;
+}
+
+export class Neo4jContainer extends GenericContainer {
+  public static readonly defaultBoltPort = 7687;
```

Comment:
any reason why not to make these private? my only thinking is that as soon as they're public, we won't be able to hide/change them - it seems they're all internal to the class

Reply:
Agree, it is better to expose less to avoid breaking changes